lra/helmcharts/sampleappslraspring-istio-openshift/Chart.yaml

```diff
@@ -16,10 +16,11 @@
 # THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
 # PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
 # CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
+ 
+ 
 apiVersion: v2
 name: sampleappslra
-description: A Helm chart for deploying MicroTx LRA Sample App in Kubernetes
+description: A Helm chart for MTRM Participants
```

Comment:
change MTRM to MicroTx

Reply:
Done, changed MTRM to MicroTx in all helm charts